README.md

```diff
@@ -41,147 +41,134 @@ _insert image of app_
 
 ### Requirements
 
-* java 1.7+
-* lein 2.5+
-* git 2.1+
-* firefox with firebug
+* Java 1.7+
+* Leiningen 2.5+
+* Git 2.1+
+* Firefox with Firebug
 
 Everything should be set up the night before during our install-fest.
 Please ensure you have everything working _before_ you show up for
 ClojureBridge on Saturday.
 
-First we're going to verify that everything is set up correctly.
+First, we're going to verify that everything is set up correctly.
 We'll do this by entering some commands in the terminal.
 
 When we demonstrate terminal commands, we'll show a snippet from our
-terminal.  Our prompt is the dollar sign, everything after the dollar
+terminal.  Our prompt is the dollar sign; everything after the dollar
 sign is the command we typed.  The result of the command is printed on
 the following lines.  Sometimes we'll have a sequence of commands in
-the same snippet, look for the dollar signs!  The terminal you're
+the same snippet — look for the dollar signs!  The terminal you're
 using will probably have different prompts but the commands should
 work the same.
 
-<div class= "console"><pre>
```

Comment:
We had those in individual div's to make clear that readers had to type three different commands.